lib/views/pages/organization/switch_org_page.dart

```diff
@@ -77,13 +77,14 @@ class _SwitchOrganizationState extends State<SwitchOrganization> {
       _successToast("Switched to ${userOrg[isSelected]['name']}");
 
       //Kill all previous stacked screen
-      Navigator.of(context).popUntil(ModalRoute.withName("/"));
+      // Navigator.of(context).popUntil(ModalRoute.withName("/"));
```

Comment:
instead of commenting, remove unwanted code.

Reply:
@Sagar2366 sir I have removed the commented code now.